scripts/extensions/pool_update.precheck

```diff
@@ -100,17 +102,10 @@ def execute_precheck(session, control_package, yum_conf_file):
         else:
             raise PrecheckFailure()
     else:
-        lines = output.splitlines()
-        resultlines = [l for l in lines if l.startswith('<livepatch ')]
-        if resultlines:
-            result = resultlines[-1].split('\"')[1]
-            try:
-                return livepatch_dic[result]
-            except KeyError:
-                raise PrecheckFailure(
-                    'Install %s, precheck error: %s' % (control_package, result))
-        else:
-            return 'ok'
+        for key in livepatch_dic.keys():
+            if key in output:
+                return livepatch_dic[key]
+        return 'ok'
```

Comment:
For livepatch, it requires to return one of the three messages.
If it doesn't return any of them, this is an invalid livepatch.
Also `ok` refers to `PATCH_PRECHECK_LIVEPATCH_NOT_APPLICABLE`, I think it makes sense.